ts-packages/react-components/package.json

```diff
@@ -30,6 +30,7 @@
     "@mui/styles": ">= 5",
     "@mui/system": ">= 5",
     "@nymproject/mui-theme": "1",
+    "zxcvbn": "^4.4.2",
```

Comment:
this plugin looks like kind of dead, last publish 6y ago (that looks like an eternity for a js package)

Reply:
this could be a better alternative https://zxcvbn-ts.github.io/zxcvbn/guide/getting-started/